model.py

```diff
@@ -208,14 +249,25 @@ def decode_image_latents(self, denoised_latent_image: Tensor) -> np.ndarray:
 
 # Singular forward pass with fake images
 # (B, 3, I, I)
-def forward_pass_test():
-    img = torch.randn([BATCH_SIZE, 3, IMG_DIM, IMG_DIM], device=DEVICE)
-    input_str = """
-    Once upon a time, in the heart of an ancient kingdom, there was a stunning garden filled with vibrant flowers and chirping birds. Amidst the garden, a glistening pond served as a mirror, reflecting the blossoming surroundings. 
-    A gentle breeze rustled the leaves of the towering trees, adding a melodious charm to the scene. The garden was indeed a paradise, a spectacle of nature's magnificence, where every creature lived in har
-    """
-    denoised_latent_img = model(img, [input_str]*BATCH_SIZE, [1]*BATCH_SIZE)
-    # print(denoised_latent_img.shape)
+@torch.no_grad()
+def forward_pass_test(model: LLourney, input_str: str, batch_size: int = 2, device="cpu"):
+    # Prepare random image in pixel space
+    img = torch.randn((batch_size, 3, model.image_dim, model.image_dim), device=device)
+
+    # Prepare input tokens and attention mask (for padding)
+    input_str_list = [input_str] * batch_size
+    input_ids, attention_mask = model.tokenize(input_str_list)
```

Comment:
I know it's used as a mask of the attention, but I think calling it `pad_mask` here is more in-line with the output from a tokenizer.